<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: blue;
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>تبریک شما برنده شدید.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 3px solid blue;
	border-radius: 5px;
	color: blue;
}

QPushButton:pressed {
	background-color: blue;
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>خرج شدن</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 3px solid blue;
	border-radius: 5px;
	color: blue;
}

QPushButton:pressed {
	background-color: blue;
	color: #fff;
}</string>
   </property>
   <property name="text">
    <string>بازی مجدد</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
